Pages/LanguagePage.cs

```diff
@@ -0,0 +1,138 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project_Mars.Pages
+{
+    public class LanguagePage
+    {
+        public void CreateLanguageRecord(IWebDriver driver, string language, string level)
```

Comment:
Please implement the POM Pattern, please see the below  for example how it looks you can see the pages how they are set 
https://medium.com/@jignect/mastering-selenium-c-with-nunit-in-depth-guide-to-page-object-model-pom-and-data-object-model-62a9c71aa01c

https://www.selenium.dev/documentation/test_practices/encouraged/page_object_models/